samples/estimation/estimation-qiskit.ipynb

```diff
@@ -590,7 +590,7 @@
       "name": "python3"
     },
     "kernelspec": {
-      "display_name": "Python 3 (ipykernel)",
```

Comment:
This display name (and python version number below) change every time I open a notebook, but they change to a different value than the one you list here. We should probably have a standard we agree on for the name and version across these notebooks. I think "Python 3" is a better display name than ".venv".

Reply:
It's going to be whatever kernel was active when the notebook was saved. If using a virutal environment, (which folks should be), then it was show that, and ".venv" is a very common pattern (e.g. the PR Ian just opened shows the same change - https://github.com/microsoft/qsharp/pull/2437/files#diff-09a59e4bf67ef253c4865a8c0a11f945967509461947ddca865017d159064bbb )

I don't think the value set here can be generic in an effective or cross platform way. Sadly, I think it's best just to ignore it.